### PR TITLE
fixed employee password email template.

### PR DESCRIPTION
**Pull request by yaniv14**

### CODE REVIEW

File: admin-dev/themes/default/sass/partials/_rtl.sass [hunk 1]
```diff
@@ -152,30 +152,28 @@ html
...
+					border-right: 14px solid $main-color
+					border-left: 0 !important
+			&.done
+				.chevron:after
+					border-right: 14px solid $brand-success
+					border-left: 0 !important
+	.actionBar
+		.msgBox
 			float: right !important
-	#carrier_logo_block
```
Comment (firstred) [inline admin-dev/themes/default/sass/partials/_rtl.sass]: Why is this one removed?

File: admin-dev/themes/default/sass/partials/_rtl.sass [hunk 2]
```diff
@@ -152,30 +152,28 @@ html
...
-		left: 10px
-		right: auto !important
-	.actionBar a
```
Comment (firstred) [inline admin-dev/themes/default/sass/partials/_rtl.sass]: Does the actionbar link style have to be removed as well?

Comment (yaniv14) [inline admin-dev/themes/default/sass/partials/_rtl.sass]: carrier_logo_block was my mistake (no need for that).
the actionbar moved above (better)

Comment (firstred) [inline admin-dev/themes/default/sass/partials/_rtl.sass]: Alrighty, makes sense

**Comment by firstred** [comment]
Thanks!

**Pull request merged**